LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -91,7 +94,7 @@ public async Task<List<IoTHubDeviceInfo>> GetDeviceList(string devEUI, string ga
                 var cacheKeyDevNonce = string.Concat(devEUI, ":", devNonce);
                 var lockKeyDevNonce = string.Concat(cacheKeyDevNonce, ":joinlockdevnonce");
 
-                if (this.cacheStore.StringSet(cacheKeyDevNonce, devNonce, TimeSpan.FromMinutes(5), onlyIfNotExists: true))
+                if (this.cacheStore.StringSet(cacheKeyDevNonce, devNonce?.ToString(), TimeSpan.FromMinutes(5), onlyIfNotExists: true))
```

Comment:
I decided to not add value-type support on the cache store, since the `devNonce` is present as `string` key already due to `string.Concat(devEUI, ":", devNonce)`